Remove form preview from the app

nmajor25 opened the pull request:
Shortcut Story ID: [sc-61636]

Currently form preview is only used in the programs admin (which doesn't get much use). And most forms are too complex to preview.

Removes all form preview related code besides what's in the `formapp.js` file. That file is under active development so that code will be removed at a later time.

This PR also includes a few miscellaneous test updates.

<!-- This is an auto-generated comment: release notes by coderabbit.ai -->

## Summary by CodeRabbit

## Summary by CodeRabbit

- **Bug Fixes**
  - Disabled form preview buttons throughout the application, making them non-interactive.
  - Removed the ability to preview forms from program flows, workflows, and sidebars.

- **Tests**
  - Deleted all tests related to form preview functionality.
  - Updated integration tests to reflect the removal of form preview and to check for disabled form preview buttons.

- **Documentation**
  - Removed localization strings related to form preview.

- **Chores**
  - Cleaned up code by removing all components, event handlers, service methods, and logic associated with form preview features.

<!-- end of auto-generated comment: release notes by coderabbit.ai -->

nmajor25 committed:
Remove form preview link button from program action sidebar

nmajor25 committed:
Remove link to form preview for program-flow action list items

nmajor25 committed:
Remove link to form preview for program workflows action list items

nmajor25 committed:
Remove remaining form preview app, view, & event router code

formapp.js preview code will remain for now, to be deleted at a future date

nmajor25 commented on src/js/views/programs/program/flow/action-item.hbs:
```diff
@@ -12,7 +12,7 @@
     <span class="table-list__meta">
       {{#if hasForm}}
-        <button class="js-form button-secondary--compact">{{far "square-poll-horizontal"}}</button>
+        <button class="button-secondary--compact" disabled>{{far "square-poll-horizontal"}}</button>
```
I switched the form icon for action list items to be `disabled`.

Instead we could also just show a form icon without button styling or keep the button enabled and open the action sidebar when the form button is clicked.

paulfalgout reviewed (changes requested):
the     'fetch:form': 'fetchForm', from the form service won't be used anymore.. so we need to strip that too.

nmajor25 committed:
Remove 'fetch:form': 'fetchForm' from form service

Code is no longer in use

nmajor25 committed:
Fix coverage for when 'patient_creation_form' org setting is not set

nmajor25 committed:
In worklist web socket tests, ensure sockets are initiated before running the test

To match what was done here: https://github.com/RoundingWell/care-ops-frontend/pull/1400/commits/aea7a9ab9c4dab28cae25e332bb81fbf94b34661

paulfalgout reviewed (approved):
Nice bit of clean up
